### PR TITLE
refs task #7627 User reading lists

### PR DESCRIPTION
**Pull request by buihanh-fr**
https://edu-redmine.framgia.vn/issues/7627

### CODE REVIEW

**Comment by buihanh-fr** [comment]
ready

File: resources/views/front/items/reading_list.blade.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+                    </button>
+                    <div class="dropdown-divider"></div>
+                    <a class="dropdown-item" target="_blank" href="https://www.facebook.com/sharer.php?u={{ $list->share_url }}"><i class="fa fa-facebook fa-fw"></i> @lang('app.share_via_facebook')</a>
+                    <a class="dropdown-item" target="_blank" href="https://twitter.com/intent/tweet?text={{ $list->share_text }}&url={{ $list->share_url}}"><i class="fa fa-twitter fa-fw"></i> @lang('app.share_via_twitter')</a>
+                    <a class="dropdown-item" target="_blank" href="mailto:?subject={{ $list->share_text }}&body={{ $list->share_text }}%0A{{ $list->share_url }}"><i class="fa fa-envelope fa-fw"></i> @lang('app.share_via_email')</a>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
```
Comment (BunnyPi04) [inline resources/views/front/items/reading_list.blade.php]: thiếu dòng trắng cuối file

File: resources/views/front/items/reading_list.blade.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+                    <i class="fa fa-ellipsis-h"></i>
+                </button>
+                <div class="dropdown-menu dropdown-menu-right">
+                    <button class="dropdown-item on-delete-list" data-delete-url="{{ route('delete_list', ['id' => $list->id]) }}">
+                        <i class="fa fa-trash fa-fw"></i>
+                        @lang('app.delete_list')
+                    </button>
+                    <div class="dropdown-divider"></div>
+                    <a class="dropdown-item" target="_blank" href="https://www.facebook.com/sharer.php?u={{ $list->share_url }}"><i class="fa fa-facebook fa-fw"></i> @lang('app.share_via_facebook')</a>
+                    <a class="dropdown-item" target="_blank" href="https://twitter.com/intent/tweet?text={{ $list->share_text }}&url={{ $list->share_url}}"><i class="fa fa-twitter fa-fw"></i> @lang('app.share_via_twitter')</a>
```
Comment (BunnyPi04) [inline resources/views/front/items/reading_list.blade.php]: convention

**Comment by buihanh-fr** [comment]
ready

File: resources/assets/frontend/css/styles.css [hunk 3]
```diff
@@ -1321,3 +1326,6 @@ ul#footer-items {
     display: block;
     color: #6f6f6f
 }
+.library .collection .info h4 a {
+    color: #222
+}
```
Comment (BunnyPi04) [inline resources/assets/frontend/css/styles.css]: thiếu dòng trắng cuối file

**Comment by buihanh-fr** [comment]
ready

**Pull request merged**